firebase-firestore/src/commonMain/kotlin/dev/gitlive/firebase/firestore/firestore.kt

```diff
@@ -27,11 +30,56 @@ expect class FirebaseFirestore {
     suspend fun clearPersistence()
     suspend fun <T> runTransaction(func: suspend Transaction.() -> T): T
     fun useEmulator(host: String, port: Int)
-    fun setSettings(persistenceEnabled: Boolean? = null, sslEnabled: Boolean? = null, host: String? = null, cacheSizeBytes: Long? = null)
     suspend fun disableNetwork()
     suspend fun enableNetwork()
 }
 
+val FirestoreSettings.Companion.CACHE_SIZE_UNLIMITED get() = -1L
+val FirestoreSettings.Companion.DEFAULT_HOST get() = "firestore.googleapis.com"
```

Comment:
I dont see this property on android?

Reply:
https://github.com/firebase/firebase-android-sdk/blob/1d717ea6c7f1feb3219b254008212e8e73b9ce18/firebase-firestore/src/main/java/com/google/firebase/firestore/FirebaseFirestoreSettings.java#L33

Reply:
https://stackoverflow.com/questions/17035271/what-does-hide-mean-in-the-android-source-code

Reply:
Ill make it internal only